### PR TITLE
List API

### PR DESCRIPTION
**Pull request by qijinze911**
Create service.py and complete list API

### CODE REVIEW

**Comment by qijinze911** [comment]
Also, error handlers are already in `service.py`.

File: service/service.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    message = str(error)
+    app.logger.error(message)
+    return jsonify(status=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                   error='Internal Server Error',
+                   message=message), status.HTTP_500_INTERNAL_SERVER_ERROR
+                   
+######################################################################
+# LIST ALL RECOMMENDATIONS
+######################################################################
+@app.route('/recommendations', methods=['HTTP GET'])
```
Comment (cheryl1223) [inline service/service.py:100]: please write test case for it before you send out PR

File: service/service.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+# variety of backends including SQLite, MySQL, and PostgreSQL
+from flask_sqlalchemy import SQLAlchemy
+from service.models import Pet, DataValidationError
+
+# Import Flask application
+from . import app
+
+######################################################################
+# Error Handlers
+######################################################################
```
Comment (cheryl1223) [inline service/service.py:46]: no need to write error handlers now, if you keep it that way, the test coverage would be poor

**Review by cheryl1223 — COMMENTED** [review]
please have your previous PR#34 merged first and send out this PR again. Please do not send out a PR without corresponding test cases.

**Pull request closed without merging**